src/screens/shared/components/NewFarmCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import '../../../styles/NewFarmCard.css';
+import fileIcon from '../../../assets/file-icon.png';
+
+class NewFarmCard extends React.PureComponent {
+  render() {
+    return (
+      <div className="newfarm-card">
```

Comment:
You don't have to address this now but I just want to clarify that this will eventually go to the new farm form?